include/seeds.dao.hpp

```diff
@@ -31,17 +33,42 @@ CONTRACT dao : public contract {
           {}
 
       name alliance_scope = "alliance"_n;
-      name campaign_scope = contracts::proposals;
+      name campaign_scope = "campaign"_n;
       name milestone_scope = "milestone"_n;
-      name referendums_scope = contracts::referendums;
+      name referendums_scope = "referendum"_n;
+      name dhos_scope = "dhos"_n;
 
       std::vector<name> scopes = {
         alliance_scope,
         campaign_scope,
         milestone_scope,
-        referendums_scope
+        referendums_scope,
+        dhos_scope
       };
 
+      name alliance_fund = "alliance"_n;
+      name campaign_fund = "campaign"_n;
+      name milestone_fund = "milestone"_n;
+
+      std::vector<name> fund_types = {
+        alliance_fund,
+        campaign_fund,
+        milestone_fund
+      };
+
+      const name prop_active_size = "prop.act.sz"_n;
```

Comment:
Are we actually keeping track of all these sizes, or are they carryover from the other contracts?

I know that we had them in the old contracts, but not sure they are still all in use.

Reply:
yes, the dao contract uses all those sizes. I am not sure all of them are in use, I think the `votepow.sz`is not used but I included anyways just to try to mimic proposals behavior. There are four sizes but only the votepow.sz is the one that I think is not needed